db/migrate/20190219180039_add_comments_to_authentications_table.rb

```diff
@@ -0,0 +1,58 @@
+class AddCommentsToAuthenticationsTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :authentications, "Authentication information for various resources."
+
+    change_column_comment :authentications, :id, "[builtin] The internal record ID. This is the primary key."
+    change_column_comment :authentications, :name, "Symbolic name for the authentication"
+    change_column_comment :authentications, :authtype, "The authentication type or role, e.g. ssh_keypair, default, amqp, etc."
+    change_column_comment :authentications, :userid, "The userid used in a userid + password authentication approach."
+    change_column_comment :authentications, :password, "The password for the given userid, if any. Encrypted."
+    change_column_comment :authentications, :resource_id, "The ID of the associated resource."
+    change_column_comment :authentications, :resource_type, "The STI type of the associated resource."
+    change_column_comment :authentications, :created_on, "[builtin] The timestamp the record was created."
+    change_column_comment :authentications, :updated_on, "[builtin] The timestamp the record was last updated."
+    change_column_comment :authentications, :last_valid_on, "The timestamp this authentication was last used successfully."
+    change_column_comment :authentications, :last_invalid_on, "The timestamp this authentication last failed."
+    change_column_comment :authentications, :credentials_changed_on, "The timestamp that credentials were last changed."
+    change_column_comment :authentications, :status, "The current status of the authentication, e.g. valid, invalid, incomplete, etc."
+    change_column_comment :authentications, :status_details, "Detailed information regarding the current status."
+    change_column_comment :authentications, :type, "The STI type of the authentication, e.g. AuthKeyPair, AuthToken, AuthUseridPassword, etc."
+    change_column_comment :authentications, :auth_key, "The private ssh key associated with the resource. Encrypted."
+    change_column_comment :authentications, :fingerprint, "The RSA key fingerprint for the SSH public key."
+    change_column_comment :authentications, :service_account
+    change_column_comment :authentications, :challenge
+    change_column_comment :authentications, :login
+    change_column_comment :authentications, :public_key, "An SSH public key."
+    change_column_comment :authentications, :htpassd_users
+    change_column_comment :authentications, :ldap_id
+    change_column_comment :authentications, :ldap_email
+    change_column_comment :authentications, :ldap_name
+    change_column_comment :authentications, :ldap_preferred_user_name
+    change_column_comment :authentications, :ldap_bind_dn
+    change_column_comment :authentications, :ldap_insecure
+    change_column_comment :authentications, :ldap_url
```

Comment:
@djberg96 I tried to shed some light on some of these but I am not sure on some. If you would like to schedule a meeting I'd be glad to  research them together.
Joe